app/assets/stylesheets/dm/pages/_search.scss

```diff
@@ -65,9 +65,9 @@
   //   margin-bottom: 2rem;
   // }
 
-  #search-results a:visited {
-    color: $dm-color-black;
-  }
+  //#search-results a:visited {
```

Comment:
nit: should we just remove styles we no longer need?

Reply:
may as well :)